src/python/WMCore/ReqMgr/DataStructs/RequestStatus.py

```diff
@@ -77,6 +77,14 @@
 
 # if the state is not defined here (new, assignment-approved) allows all the property to get 
 ALLOWED_ACTIONS_FOR_STATUS = {
+                 "assignment-approved":["Team", "SiteWhitelist", "SiteBlacklist",
+                                        "AcquisitionEra", "ProcessingString", "ProcessingVersion", 
+                                        "Dashboard", "MergedLFNBase", "MaxRSS", "TrustSitelists", 
+                                        "UnmergedLFNBase", "MinMergeSize", "MaxMergeSize",
+                                        "MaxMergeEvents", "MaxVSize", 
+                                        "BlockCloseMaxWaitTime", 
+                                        "BlockCloseMaxFiles", "BlockCloseMaxEvents", "BlockCloseMaxSize",
+                                        "SoftTimeout", "GracePeriod"],
```

Comment:
For this assignment-approved filter, we have to add:
TrustPUSitelists, CustodialSites, CustodialSubType, NonCustodialSites, NonCustodialSubType, AutoApproveSubscriptionSites, SubscriptionPriority

I'd also suggest to organize these parameters such that they are next to each other in the GUI (like, merged and unmerged).

In addition to that, we need to handle list or dict values on the server side. I believe the javascript input will be a string right?